nomad/leader.go

```diff
@@ -992,6 +995,24 @@ func (s *Server) reapDupBlockedEvaluations(stopCh chan struct{}) {
 	}
 }
 
+// reapCancelableEvaluations is used to reap evaluations that were marked
+// cancelable by the eval broker and should be cancelled. These get swept up
```

Comment:
```suggestion
// cancelable by the eval broker and should be canceled. These get swept up
```

literally impossible to remember to be consistent with this spelling... stupid english